cx_Freeze/windist.py

```diff
@@ -296,7 +297,21 @@ def add_files(self):
         while todo:
             dir = todo.pop()
             for file in os.listdir(dir.absolute):
-                if os.path.isdir(os.path.join(dir.absolute, file)):
```

Comment:
@Cimbali Apparently this change is the root cause of issue #1184 
Can you take a look?